spec/models/mixins/assignment_mixin_spec.rb

```diff
@@ -78,6 +78,22 @@
     end
   end
 
+  describe "#get_assigned_tos" do
```

Comment:
added test because once this exposed and available in API, would not want format to change